contractsV5/KyberNetwork.sol

```diff
@@ -106,8 +98,8 @@ contract KyberNetwork is Withdrawable, Utils, IKyberNetwork, ReentrancyGuard {
     /// @dev add or deletes a reserve to/from the network.
     /// @param reserve The reserve address.
     function addReserve(IKyberReserve reserve, uint reserveId, bool isFeePaying) public onlyOperator returns(bool) {
-        require(reserveIdToAddresses[reserveId][0] == address(0));
-        require(reserveAddressToId[address(reserve)] == uint(0));
+        require(reserveIdToAddresses[reserveId].length == 0, "reserve id already taken");
```

Comment:
current plan is that each reserve ID
has current reserve in cell 0
and others in next cells

I assume the remove reserve will remove the one in array cell 0 and move it to last array cell.
cell 0 address will be set to address(0)
this is why I make sure cell 0 with address(0)
so since its mapping array of addresses. length will be 0 only for first address per ID